var/spack/repos/builtin/packages/lua/package.py

```diff
@@ -58,24 +58,31 @@ class Lua(Package):
         destination="luarocks",
         placement='luarocks')
 
+    patch('https://git.archlinux.org/svntogit/packages.git/plain/trunk/liblua.so.patch?h=packages/lua&id=70abed4cfa74993348bf7afbbc70151a3d57cc01',
+            sha256='2cc83c77423a2dda3696766b2d1ccee2796e052ab04d5178905f41ed9241a3d8')
+
     def install(self, spec, prefix):
         if spec.satisfies("platform=darwin"):
             target = 'macosx'
         else:
             target = 'linux'
         make('INSTALL_TOP=%s' % prefix,
+             'MYCFLAGS=-fPIC',
```

Comment:
here and below please use `self.compiler.pic_flag`

Reply:
Done. :-)